ghub.py

```diff
@@ -86,6 +86,13 @@ def https_open(self, req):
 urllib.request.install_opener(opener)
 
 
+def display_colored_state(state):
```

Comment:
A couple minor nits--could we name this state_color instead, since it's not displaying anything, just returning a color? And maybe make it a bit more compact, something like:
```python
def state_color(state):
    return {'open': 'green', 'closed': 'red'}.get(state.lower())
```

Reply:
Sure thing @ccstolley . Changes made!